charts/secrets-injector/README.md

```diff
@@ -0,0 +1,38 @@
+# 1Password Kubernetes Secrets Injector Helm chart
+Installing the Helm Chart with default configurations will deploy 1Password Kubernetes Secrets Injector in your default Namespace.
+
+## Installation
+```
+helm install --generate-name 1password/secrets-injector
+```
+
+## Usage
+Please follow [Secrets Injector documentation](https://github.com/1Password/kubernetes-secrets-injector#1password-secrets-injector-for-kubernetes)
```

Comment:
This feel a bit too small to include it as a section.

We have 2 options here:
- Remove it altogether (I see we don't include this for Connect Helm Charts)
- Rephrase it to make it fit better. Maybe something like this:
  ```suggestion
  ## Usage
  Check out the [documentation for the Secrets Injector](https://github.com/1Password/kubernetes-secrets-injector#1password-secrets-injector-for-kubernetes) to see how to use it in your infrastructure.
  ```

Reply:
I removed it